.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: Gradle Build
+
+on:
+  push:
+    branches: [ trunk ]
+  pull_request:
+    branches: [ trunk ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
```

Comment:
Suggestion: Maybe call this build and test.

Reply:
Good call. we updated it here [c7c1102](https://github.com/jramnani/GildedRoseJavaLLC/pull/20/commits/c7c1102f0ed51117e61d06f3424cb319396fd082)